migrations/entitlements/migration.go

```diff
@@ -131,6 +131,13 @@ func ConvertValueToEntitlements(
 			referenceValue.Authorization,
 			interpreter.ConvertSemaToStaticType(inter, referenceValue.BorrowedType),
 		)
+	case interpreter.LinkValue: //nolint:staticcheck
+		// Link values are not supposed to reach here.
+		// But it could, if the type used in the link is not migrated,
+		// then the link values would be left un-migrated.
```

Comment:
Is this a possible result of the migration?

Reply:
yeah, for e.g: if type-loading fails during link-migration, then the link values currently remains in the storage as-is.